dotCMS/src/main/java/com/dotcms/graphql/InterfaceType.java

```diff
@@ -114,7 +132,10 @@ public GraphQLInterfaceType getType() {
 
         for(final InterfaceType type : InterfaceType.values()) {
             if(type.getType()!=null) {
-                types.add(type.getType());
+                if(!EnterpriseType.class.isAssignableFrom(type.baseContentType)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [These nested if statements could be combined](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4740432)